### PR TITLE
Creating gradle task for generating environment bundles

### LINKED ISSUE #153: Enable an environment bundle to be built by calling a task in the gradle plugins
Create a gradle task to build the environment bundle, using specified properties from gradle config.

child of #140

### PR DESCRIPTION
**Pull request by joao-borges**
Fixes #153 

## Proposed Changes
* created task `build-environment-bundle` that expects environment config provided in the gradle file and generate the environment bundle

### CODE REVIEW

File: config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/EntityLoader.java [hunk 1]
```diff
@@ -9,8 +9,11 @@
 import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
 
 import java.io.File;
+import java.util.Map;
```
Comment (apigatewayopensource) [inline config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/EntityLoader.java]: Code Smell Code Smell: Remove this unused import 'java.util.Map'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=154&issues=AWekvNyWh8YERCevmIXA&open=AWekvNyWh8YERCevmIXA)

[ISSUE_KEY]: AWekvNyWh8YERCevmIXA

**Comment by yves-chan** [comment]
Hey Joao, do you think it's worth making a test in CAGatewayDeveloperTest or creating a BuildEnvironmentBundleTaskTest class?

**Comment by joao-borges** [comment]
@yves-chan all the new methods in the CAGatewayDeveloperTest are covering this new implementations.

**Comment by yves-chan** [comment]
@joao-borges ah you're right, I see it. I think I was looking at an old PR :)

**Pull request merged**